### PR TITLE
feat(leaderboards): add GDP chart

### PR DESCRIPTION
**Pull request by Akuukis**
resolves https://github.com/benreid24/Stellaru/issues/95

### CODE REVIEW

File: frontend/src/Monitor/Leaderboard/Charts/GDP.tsx
Comment (benreid24) [inline frontend/src/Monitor/Leaderboard/Charts/GDP.tsx:34]: We should remove this logging before merging

File: frontend/src/Monitor/Leaderboard/Charts/GDP.tsx
Comment (benreid24) [inline frontend/src/Monitor/Leaderboard/Charts/GDP.tsx:35]: It might be nice to add dropdowns to allow the user to choose between the different gdp types (`net` vs `gross` and `base value` vs `market value`)

**Review by benreid24 — COMMENTED** [review]
This looks great! Let's just remove the data logging and it's good to go. I'll leave it up to you if you want to add dropdowns for allowing the user to select what data is actually charted.

File: frontend/src/Monitor/Leaderboard/Charts/GDP.tsx
Comment (Akuukis) [reply frontend/src/Monitor/Leaderboard/Charts/GDP.tsx:34]: agree, for later. I added the dropdown to VIctory and it looks bad. Here it's optional (at least of my needs), so prefer to keep it simple and beautiful.

**Review by benreid24 — APPROVED** [review]

**Pull request merged**
